subsys/bluetooth/Kconfig.iso

```diff
@@ -35,7 +51,6 @@ config BT_ISO_SYNC_RECEIVER
 	select BT_ISO_BROADCAST
 	select BT_OBSERVER
 	select BT_PER_ADV_SYNC
-	select EXPERIMENTAL
```

Comment:
You shouldn't remove this unless you also remove "[EXPERIMENTAL]" from the option description

Reply:
Experimental is now added to the parent BT_ISO instead, that should be ok, right?

Reply:
I thought "[EXPERIMENTAL]" in the option description and `select EXPERIMENTAL` always went hand-in-hand. @carlescufi can you confirm if this is the case or not?

Reply:
I don't mind adding it. 

It's a good question: Should we also have `select EXPERIMENTAL` and `[EXPERIMENTAL]` for hidden Kconfigs, or just for those selectable?

Reply:
Updated the Kconfigs such that we keep a 1:1 between `[EXPERIMENTAL]` and `select EXPERIMENTAL`, effectively removing it from the hidden Kconfigs and adding them to the selectable ones.